src/components/shared/VirtualList.js

```diff
@@ -386,7 +386,17 @@ export class VirtualList<Item> extends React.PureComponent<
     const itemBottom = itemTop + this.props.itemHeight;
     const itemBottomWithMargin = itemBottom + scrollMargin;
 
-    if (itemTopWithMargin < container.scrollTop) {
+    const bigJump = 16 * this.props.itemHeight;
+    if (
+      itemTop + bigJump < container.scrollTop ||
+      itemBottom - bigJump > container.scrollTop + container.clientHeight
+    ) {
+      const scrollTopToCenterItem =
```

Comment:
nit: it would ne nice to explain why we have this case in a comment for future reference.

Reply:
Good idea, thanks!